build_openstudio_gems.rb

```diff
@@ -136,7 +136,7 @@ def make_package(install_dir, tar_exe, expected_ruby_version, bundler_version)
   lib_ext = RbConfig::CONFIG['LIBEXT']
   libs = Dir.glob("./openstudio-gems/**/*.#{lib_ext}")
   lib_names_woext = Set.new(libs.map{|lib| File.basename(lib, File.extname(lib)) })
-  expected = Set.new(["jaro_winkler_ext", "libll", "liboga", "msgpack", "byebug"]) # unf_ext: disabled with json_schemer
+  expected = Set.new(["jaro_winkler_ext", "libll", "liboga", "msgpack", "byebug", "generator", "parser"]) # unf_ext: disabled with json_schemer
```

Comment:
I think you're missing changes in https://github.com/NREL/openstudio-gems/blob/45b80c5c499590e7a62ba90359eb9b66c2b562c3/lib/rubygems_plugin.rb#L92-L96

Reply:
```
Warning: rubygems_plugin.post_install: no configuration given for extension_dir=/home/runner/work/openstudio-gems/openstudio-gems/openstudio-gems/ruby/3.2.0/gems/json-2.7.2/ext/json/ext/generator
Warning: rubygems_plugin.post_install: no configuration given for extension_dir=/home/runner/work/openstudio-gems/openstudio-gems/openstudio-gems/ruby/3.2.0/gems/json-2.7.2/ext/json/ext/parser
```

https://github.com/NREL/openstudio-gems/actions/runs/8974770681/job/24647856284?pr=87#step:8:464

Reply:
You'll also need some changes on the SDK side